src/app/admin/admin-reports/filtered-items/filtered-items.component.ts

```diff
@@ -293,6 +291,14 @@ export class FilteredItemsComponent {
       );
   }
 
+  /*postFilteredItems(): Observable<RawRestResponse> {
```

Comment:
@jeffmorin : Could we remove this commented out code?  It appears to have be added accidently?

Reply:
Done. Sorry about that!